docs/DSP/GameCube_DSP_Users_Manual/GameCube_DSP_Users_Manual.tex

```diff
@@ -704,7 +704,7 @@ \section{Hardware registers}
 \Address{0xFFDA} & \Register{SCALE}   & ADPCM predictor and scale     \\ \hline
 \Address{0xFFDB} & \Register{YN1}     & ADPCM YN1                     \\ \hline
 \Address{0xFFDC} & \Register{YN2}     & ADPCM YN2                     \\ \hline
-\Address{0xFFDD} & \Register{ACDAT}   & Accelerator data              \\ \hline
+\Address{0xFFDD} & \Register{ACDSAMP} & Accelerator processed sample  \\ \hline
 \Address{0xFFDE} & \Register{GAIN}    & Gain                          \\ \hline
 \Address{0xFFDF} & \Register{ACUNK2}  & Unknown, usually \Value{0x0C} \\ \hline
```

Comment:
Also change this line as per line 415 of `DSPTables.cpp`?

Reply:
Yeah sorry I'm intending to do a documentation sweep once everything else is settled. It's pretty close now.

Reply:
That's fine :) Just for mention that